Error when start MQTTany

when i start MQTTany get i this fault

```log
2020-12-30 14:27:33,770 [ERROR] [core                    ] 

Traceback (most recent call last):
  File "mqttany/mqttany.py", line 110, in <module>
    core.start(queue_core, args.config_file)
  File "/opt/mqttany/mqttany/core.py", line 382, in start
    import gpio
  File "/opt/mqttany/mqttany/gpio/__init__.py", line 47, in <module>
    board = get_board()
  File "/opt/mqttany/mqttany/gpio/boards/__init__.py", line 56, in get_board
    return all_boards.get(Detector().board.id, Unknown)()
  File "/opt/mqttany/mqttany/gpio/boards/__init__.py", line 42, in __init__
    self._chips.append(999)  # dummy
AttributeError: 'Unknown' object has no attribute '_chips'

2020-12-30 14:27:33,770 [WARN ] [core                    ] MQTTany exiting with errors
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,9 @@
 
 ## Development
 
+* **Fixed**
+  * *Unknown* and *Generic Linux* boards missing superclass init.
+
 ## 0.13.0
 
 * ***BREAKING CHANGE***

--- a/mqttany/gpio/boards/__init__.py
+++ b/mqttany/gpio/boards/__init__.py
@@ -38,12 +38,14 @@ from .base import Board
 
 class Unknown(Board):
     def __init__(self) -> None:
+        super().__init__()
         self._id = "UNRECOGNIZED"
         self._chips.append(999)  # dummy
 
 
 class Generic(Board):
     def __init__(self) -> None:
+        super().__init__()
         self._id = GENERIC_LINUX_PC
         self._chips.append(999)  # dummy
 

